apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx

```diff
@@ -425,9 +456,8 @@ export const AttributeSelectionOIDC: FunctionComponent<AttributeSelectionOIDCPro
                         requested: claim.requested
                     };
 
-                    if (!initialSelectedClaims.find(
-                        (selectedExternalClaim) => selectedExternalClaim?.mappedLocalClaimURI
-                        === claimMapping.localClaim.uri)){
+                    if (!initialSelectedClaims.find((selectedExternalClaim: ExtendedExternalClaimInterface) => 
+                        selectedExternalClaim?.mappedLocalClaimURI === claimMapping.localClaim.uri)){
```

Comment:
Can we change like this? Please check the other places also. :)
```suggestion
                        selectedExternalClaim?.mappedLocalClaimURI === claimMapping?.localClaim?.uri)){
```